enigma-core/enclave/src/auto_ffi.rs

```diff
@@ -1,8 +1,8 @@
-// automatically generated by rust-bindgen
+cargo:warning=couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
```

Comment:
Please remove this from the PR. The file's contents didn't change, but there wos a warning during the generation.

Reply:
Done